_pages/plugins/modularimageanalysis.md

```diff
@@ -0,0 +1,217 @@
+4. It's also possible to allow modules to be disabled from processing view by selecting "Can be disabled" from the top of the parameter window.
+    - When selected, a power icon ( or ) will be displayed next to the module name in processing view.
+5. Modules and parameters can be renamed by right-clicking on their name.
+
+### Configuring output results format
+1. The generated results spreadsheet can be configured in the always-present [Output control](https://mianalysis.github.io/mia/html/modules/core/outputcontrol.html) module, which can be found above the current workflow.
+    - Results spreadsheets will typically be stored in the same folder as the selected input file or within a selected input folder.
+2. By default, all measurements will be included in the spreadsheet. These can be enabled/disabled individually from the "Measurement selection" section.
+    - Each exported object collection will be given a separate sheet in the worksheet.
+    - Measurement names, which will be displayed at the top of each column, can be changed by right-clicking on the measurement name.
+3. Measurements will also be summarised in a separate "Summary" sheet. This sheet can be enabled/disabled in the "Summary sheet" section.
+    - A single summary is prepared for each processed image; however, this can be split down per timepoint.
+    - Each measurement can be summarised as the mean, minimum, maximum, standard deviation and sum.
+    - Individual measurement summaries can be enabled/disabled in the "Measurement selection" section.
+
+## Acknowledgements
+The plugin makes use of a combination of plugins packaged with Fiji as well as others that can be installed via the updater.
+
+Required plugins pre-packaged with Fiji: [AnalyzeSkeleton](/plugins/analyze-skeleton), [Auto Threshold](/plugins/auto-threshold), [bUnwarpJ](/plugins/bunwarpj/index), [BioFormats](/formats/bio-formats), [Colour Deconvolution](/plugins/colour-deconvolution), [Correct Bleach](/plugins/bleach-correction), [Feature Extraction](/plugins/feature-extraction), [TrackMate](/plugins/trackmate/index), [Weka Trainable Segmentation](/plugins/tws/index).  Required plugins that need installing via the Fiji updater: [MorphoLibJ](/plugins/morpholibj).  Plugins bundled with MIA: [Stack Focuser](https://imagej.nih.gov/ij/plugins/stack-focuser.html).
+
+A list of bundled dependencies along with their respective licenses can be found [here](https://htmlpreview.github.io/?https://github.com/mianalysis/mia/blob/master/target/site/dependencies.html).
+
+Special thanks to all MIA users who have provided vital feedback over the years.  In particular, big thanks to Dr. Dominic Alibhai for his many suggestions and ideas.
+
+## Citing MIA
+We hope you find MIA useful.  If you've used MIA in your research, please cite it using the Zenodo DOI for that version of the plugin.  Zenodo DOIs for MIA can be found [here](https://doi.org/10.5281/zenodo.1201320).
```

Comment:
You can [use `{% include citation %}`](https://imagej.net/editing/citations#citing-by-wiki-page-url) here to render the DOI from the front matter as a reference.